dev/com.ibm.ws.jaxws.2.3.common/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -0,0 +1,26 @@
+###############################################################################
+# Copyright (c) 2024 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+###############################################################################
+# -------------------------------------------------------------------------------------------------
+#CMVCPATHNAME com.ibm.ws.build.example.project/resources/OSGI-INF/l10n/metatype.properties
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+
+webserviceClient=JAX-WS web service client configuration
+webserviceClient.desc=Configuration for JAX-WS clients.
+
+serviceName=Web service client name
+serviceName.desc=The serviceName property defined in the @WebServiceClient annotation.
```

Comment:
Should the description say "The name of the Web service defined in the @WebServiceClient annotation"? I don't think there is a property "serviceName" defined in the annoation.